shared/design/src/stories/textField.stories.tsx

```
@@ -0,0 +1,172 @@
+  args: {
+    name: "example-error",
+    state: "error",
+  },
+  render: (args) => {
+    const [val, setVal] = useState("")
+    return (
+      <TextField.Root {...args}>
+        <TextField.Label>이메일</TextField.Label>
+        <TextField.InputWrapper>
+          <TextField.Input
+            placeholder="이메일을 입력하세요"
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)